recaptcha_enterprise/demosite/src/main/resources/static/scripts/global.js

```diff
@@ -0,0 +1,66 @@
+// Copyright 2023 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+// ATTENTION: reCAPTCHA Example (Part 3) Starts
+// SEE: https://cloud.google.com/recaptcha-enterprise/docs/create-assessment
+// SEE: If using a library or famework, can use event handlers its usual way
+
+function createAssessment({ action, token }) {
```

Comment:
Why do these functions need to be wrapped in functions?

Reply:
This is done keeping in mind to demonstrate the best practices of integrating reCAPTCHA. It mimicks the normal webapp flow even though the code here is minimal. 
We didn't want to concise everything into a single method as we intend to distinguish the different steps involved in the integration. Please let us know wdyt?